pkg/i18n/polish.go

```diff
@@ -37,6 +37,10 @@ func polishTranslationSet() TranslationSet {
 		LcScroll:                            "przewiń",
 		LcAbortMerge:                        "o scalaniu",
 		LcResolveMergeConflicts:             "rozwiąż konflikty scalania",
+		LcCommitFileFilter:                  "Filtrowanie plików commitować",
+		FilterStagedFiles:                   "Show only staged files",
```

Comment:
I don't know any polish and therefore can't verify if deepl would do a good job at translating this :D

Reply:
Best effort attempts are fine